### DISCUSSION TITLE
ImageGlass Kobe 8.6.6.6

### DISCUSSION THREAD

**Discussion by d2phap** [d2phap/ImageGlass#1351] [General]
<img width="640" alt="ImageGlass 8.6.6.6" src="https://github.com/ImageGlass/config/raw/main/screenshots/v8.6/8.6_1.jpg">

[![Downloads](https://img.shields.io/github/downloads/d2phap/imageglass/8.6.6.6/total?color=%239C613B&label=downloads&style=for-the-badge)](https://imageglass.org/download)

## 👐🏿 Support ImageGlass on Microsoft Store!
This is the paid version, you will need to pay a small amount after 7-day trial. If you buy ImageGlass in the Microsoft Store, you'll be supporting its development directly. You will get the convenience of fast, easy installation onto all of your Windows devices along with fully automatic, behind-the-scenes updates with all the newest features, improvements, and fixes.

<a href="https://www.microsoft.com/store/productId/9N33VZK3C7TH">
  <img height="65" src="https://upload.wikimedia.org/wikipedia/commons/f/f7/Get_it_from_Microsoft_Badge.svg" />
</a>

### 🌱 New:
- Added support for WEBP animated format (#508), FITS, MJPEG, VIFF, VX.
- Added a new feature on Windows 10 or later: Share (<kbd>S</kbd>).
- Added new settings for vertical / horizontal panning speed in Settings > Image > Panning, kudo to @siblount (#1179).
- Added a setting for automatically displaying new image in folder in Settings > Image > Image loading (#1348).
- Changed the URL for Update service: Since v8.6, when checking for an update, ImageGlass connects to https://imageglass.org/url/update instead of https://imageglass.org/checkforupdate.

### 🔮 Improvements:
- Updated Magick.NET to v11.1.2.
- Color management: Add option `Use current monitor profile` to auto-select the current monitor profile in Settings > Image > Color management (#781). If the monitor does not have a profile, `sRGB` will be used.
- Added support for auto-resizing the windows in Frameless mode when user zoom the viewing image, kudo to @SylwesterZarebski (#822).
- Added support for copy & paste image with transparency (#1305).
- Removed the ugly top and left border of the menu dropdown on Windows 10 (#1218).
- Improved edit feature: try to find and use MS Paint as the default editing app on Windows 11 (#1307).
- Updated the text for clearer explanation in Settings > General > Others (#1325):
  - Start with OS to improve startup speed.
  - Continue running in the background when the app is closed.

### 🐛 Bugfixes:
- Fixed an issue that the app window does not activate when opening an image from Explorer in single instance mode (#620).
- Fixed an issue when the window in inactive, user need to click twice for the toolbar buttons, kudo to @Sxderp (#1202).
- Fixed an issue when printing the viewing image crashes the app if the Temp folder is not found (#1343).
- Fixed an issue when the 32-bit MSI installer has wrong default installation directory and creates unnecessary folder (#1265).

### 🥒 Checksums:

File | SHA-1
-- | --
ImageGlass_Kobe_8.6.6.6_x64.msi | `B6CE0AC63E56C8FA4E995C926D60181A01D2F6C6`
ImageGlass_Kobe_8.6.6.6_x64.zip | `8EE32EC15D2135E3A69BA4916D01908BC450CCE5`
ImageGlass_Kobe_8.6.6.6_x86.msi | `84C513F1C311B8314DC26CF6832D98AB4B05F441`
ImageGlass_Kobe_8.6.6.6_x86.zip | `2299838ABDB05411C2F231B429AF61E2C0135537`

## 🛣 ImageGlass 9.0 development updates
- [Version 9.0 beta 1](https://github.com/d2phap/ImageGlass/releases/tag/9.0.1.514) was already released. Try it and let me know your feedback.
- [Version 9.0 beta 2](https://github.com/d2phap/ImageGlass/projects/12) is under development.
- Since version 9.0, .NET 6 Runtime is required to run ImageGlass, you can download it at https://dotnet.microsoft.com/en-us/download/dotnet/6.0.

<hr /><em>This discussion was created from the release <a href='https://github.com/d2phap/ImageGlass/releases/tag/8.6.6.6'>ImageGlass Kobe 8.6.6.6</a>.</em>

**Comment by ghost** [comment]
Please keep only ImageGlass **Remove Kobe Word** from Title Bar

![Screenshot (286)](https://user-images.githubusercontent.com/98110551/171881999-8137d866-3f66-475c-96d6-cc02cb488d49.jpg)

**Reply by d2phap** [reply]
I replied here: https://github.com/d2phap/ImageGlass/issues/1352#issuecomment-1146512884

**Comment by d2phap** [comment]
### Issue:
For the 64-bit MSI installer, if you select install type "Only for me", you may encounter a warning about `Could not write value ImageGlass Kobe to key...`.

This is because a bug of Advanced Installer 19.5, it does not trigger the admin popup dialog, and writing registry keys requires admin permission. 

![image](https://user-images.githubusercontent.com/3154213/171978388-523a6392-011b-44b1-9ac3-bdb3d787e1da.png)


### Solution:
You can click `Ignore` button to ignore this error, or choose "Everybody" instead of "Only for me".

![image](https://user-images.githubusercontent.com/3154213/171978635-001645c9-7977-422a-aa2e-62415862eacc.png)

**Reply by giwrgosmant** [reply]
> ### Issue:
> 
> For the 64-bit MSI installer, if you select install type "Only for me", you may encounter a warning about `Could not write value ImageGlass Kobe to key...`.
> 
> This is because a bug of Advanced Installer 19.5, it does not trigger the admin popup dialog, and writing registry keys requires admin permission.
> 
> ### Solution:
> 
> You can click `Ignore` button to ignore this error, or choose "Everybody" instead of "Only for me".
> 

@d2phap 
System: Windows 11
Ignore button doesn't work, it pop up all the time. I don't wanna Install it for "Everybody", is there any upcoming soom fix for it?
Thanks for your work 👍

**Reply by d2phap** [reply]
@giwrgosmant I downgraded Advanced Installer from v19.5 to v19.1, the issue is fixed now, but I'm waiting for Magick.NET team to push the new update this weekend to fix this https://github.com/d2phap/ImageGlass/issues/1354.

So probably I will push a new release this weekend!
